src/scores/categorical/contingency_impl.py

```diff
@@ -117,27 +144,66 @@ def bias_score(self):
         """
         How did the forecast frequency of "yes" events compare to the observed frequency of "yes" events?
 
-        https://www.cawcr.gov.au/projects/verification/#BIAS
+        Returns:
+            An xarray object containing the bias score
+
+        .. math::
+            \\text{frequency bias} = \\frac{\\text{true positives} + \\text{false positives}}{\\text{true positives} + \\text{false negatives}}
+
+        Notes:
+
+            - Range: 0 to ∞ (infinity), where 1 indicates a perfect score
+            - "True positives" is the same as "hits"
+            - "False positives" is the same as "false alarms"
+            - "False negatives" is the same as "misses"
+
+        References:
+            https://www.cawcr.gov.au/projects/verification/#BIAS
         """
         return self.frequency_bias()
 
     def hit_rate(self):
         """
-        What proportion of the observed events where correctly forecast?
         Identical to probability_of_detection
-        Range: 0 to 1.  Perfect score: 1.
 
-        https://www.cawcr.gov.au/projects/verification/#POD
+        Calculates the proportion of the observed events that were correctly forecast.
+
+        Returns:
+            An xarray object containing the hit rate
+
+        .. math::
+            \\text{hit rate} = \\frac{\\text{hits}}{\\text{hits} + \\text{misses}}
+
+        Notes:
+
+            - Range: 0 to 1.  Perfect score: 1.
+            - "Hits" is the same as "true positives"
+            - "Misses" is the same as "false negatives"
+
+        References:
+            https://www.cawcr.gov.au/projects/verification/#POD
         """
         return self.probability_of_detection()
 
     def probability_of_detection(self):
         """
-        What proportion of the observed events where correctly forecast?
-        Identical to hit_rate
-        Range: 0 to 1.  Perfect score: 1.
+        Identical to hit rate
 
-        https://www.cawcr.gov.au/projects/verification/#POD
+        Calculates the proportion of the observed events that were correctly forecast.
+
+        Returns:
+            An xarray object containing the probability of detection
+
+        .. math::
+            \\text{hit rate} = \\frac{\\text{hits}}{\\text{hits} + \\text{misses}}
```

Comment:
Perhaps make everything "true positives" etc rather than "hits" etc so that it's consistent throughout